pages/specification.md

```diff
@@ -8,17 +8,44 @@ redirect_from:
+## Contribute
 
-It is suggested that you present your proposals at the relevant calls and communication channels, which include the following weekly recurring meetings: 
-* Mondays at 1600 CEST Authentication Panel on [this line](https://inrupt.my.webex.com/inrupt.my/j.php?MTID=m131f1588eb27afc298276f1cbf077a79)
-* Mondays at 2130 CEST Interoperability Panel on [this line](https://global.gotomeeting.com/join/620786365) 
-* Wednesday at 1600 CEST Authorization Panel on [this line](https://global.gotomeeting.com/join/544111573)
 
-You can read a [record of call agendas and minutes](https://www.w3.org/community/solid/wiki/Meetings). 
+<div class="message is-info">
+  <p class="message-body">
+    Before contributing, please read the 
+    <a href="https://github.com/solid/process/blob/master/code-of-conduct.md">Code of Conduct</a> 
+    and <a href="https://www.w3.org/community/solid/"> join the W3C Solid Community Group</a>. 
+  </p>
+</div>
 
-For suggestions to be approved they go through an [Editorial review process](https://github.com/solid/process). 
+Anyone can contribute by filing an [issue via GitHub](https://github.com/solid/specification/issues) 
+or by submitting [pull requests](https://github.com/solid/specification/pulls).
 
-You can keep up to date with Solid specification progress by reading the [monthly updates by the Editors](https://www.w3.org/community/solid/wiki/Main_Page) which you can also receive by signing up to [This Month in Solid](https://solidproject.org/newsletter). 
+For new proposals, present your proposals at the relevant meetings 
+and communication channels, which include the following weekly recurring meetings: 
+
+| Panel | When | Meeting Link |
+| ----- |------|--------------|
+| [Authentication Panel](https://www.w3.org/community/solid/wiki/Authentication_Panel)   | Mondays at 1600 CEST   | [Meeting](https://inrupt.my.webex.com/inrupt.my/j.php?MTID=m131f1588eb27afc298276f1cbf077a79) |
+| [Interoperability Panel](https://www.w3.org/community/solid/wiki/Interoperability_Panel) | Mondays at 2130 CEST   | [Meeting](https://global.gotomeeting.com/join/620786365) |
+| [Authorization Panel](https://www.w3.org/community/solid/wiki/Authorization_Panel)   | Wednesday at 1600 CEST | [Meeting](https://global.gotomeeting.com/join/544111573) |
+
+Submitted proposals undergo an Editorial review process. For more information
+on proposal submissions and reviews, see the 
+[Solid Panel proposal process](https://github.com/solid/process#reviewing-proposals).
+
+## Track Progress
+
+To keep up-to-date with Solid specification progress, you can:
+
+* Read the [monthly updates by the Editors](https://www.w3.org/community/solid/wiki/Main_Page). 
```

Comment:
I used the same link that was here before - but this link doesn't have anything about monthly updates.

Reply:
I ended up removing that for now since that isn't how we've been communicating updates of late - in the next iteration of the page we can retool this further 🙂

Reply:
Sounds good 👍